ref/Microsoft.Build.Framework/netstandard/Microsoft.Build.Framework.cs

```diff
@@ -354,8 +354,6 @@ public partial interface ITaskItem2 : Microsoft.Build.Framework.ITaskItem
     }
     public partial class LazyFormattedBuildEventArgs : Microsoft.Build.Framework.BuildEventArgs
     {
-        [System.NonSerializedAttribute]
```

Comment:
I know that something in these assemblies implements a custom log message

https://github.com/dotnet/msbuild/blob/37c61113335898d83fcae3203846402f2af31bf1/src/MSBuild/app.config#L120-L131

I don't know if that implements `LazyFormattedBuildEventArgs` or something else. Can you run this through a VS insertion to double-check that this doesn't break that use? If it doesn't, I think #1675 will mean that this "breaking" change isn't a problem.

Reply:
It's probably [this class](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=/src/edev/StaticAnalysis/fxcoptask/CodeAnalysisErrorEventArgs.cs) and [this class](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=/src/edev/StaticAnalysis/fxcoptask/CodeAnalysisWarningEventArgs.cs). They indeed indirectly derive from `LazyFormattedBuildEventArgs` and I don't see references to the `locker` field so we should be fine. Waiting for an experimental insertion just in case.

Reply:
No issues found in the experimental insertion.